mysite/polls/models.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import datetime
+from django.db import models
+from django.utils import timezone
+from usermanager.models import PollUser
+
+class Question(models.Model):
+
+    def __str__(self):
+        return self.question_text
+
+    def was_published_recently(self):
+        return self.pub_date >= timezone.now() - datetime.timedelta(days=1)
+
+    question_text = models.CharField(max_length=200)
+    pub_date = models.DateField('date published')
+
+
+class Choice(models.Model):
+
+    def __str__(self):
+        return self.choice_text
+
+    question = models.ForeignKey(Question, on_delete=models.CASCADE)
+    choice_text = models.CharField(max_length=200)
+    votes = models.IntegerField(default=0)
+
+# Create your models here.
+class Answer(models.Model):
+
+    user_id = models.ForeignKey(PollUser, on_delete=models.CASCADE)
+    choice_id = models.IntegerField()
```

Comment:
Choice should be foreign key. You can use `choice` instead of`choice_id`,  and follow same convention for all reference keys.